packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -389,56 +406,32 @@ const NotificationDetail = () => {
     handleCancelNotification();
   };
 
-  const ModalAlert = () => (
-    <Dialog
-      open={showModal}
-      data-testid="modalId"
-      onClose={handleModalClose}
-      aria-labelledby="dialog-title"
-      aria-describedby="dialog-description"
-    >
-      <DialogTitle id="dialog-title" sx={{ p: 4 }}>
-        {t('detail.cancel-notification-modal.title', { ns: 'notifiche' })}
-      </DialogTitle>
-      <DialogContent sx={{ px: 4, pb: 4 }}>
-        <DialogContentText id="dialog-description">
-          {t('detail.cancel-notification-modal.message', { ns: 'notifiche' })}
-        </DialogContentText>
-      </DialogContent>
-      <DialogActions sx={{ px: 4, pb: 4 }}>
-        <Button onClick={handleModalClose} variant="outlined" data-testid="modalCloseBtnId">
-          {t('button.indietro')}
-        </Button>
-        <Button
-          onClick={handleModalCloseAndProceed}
-          variant="contained"
-          data-testid="modalCloseAndProceedBtnId"
-        >
-          {t('new-notification-button', { ns: 'notifiche' })}
-        </Button>
-      </DialogActions>
-    </Dialog>
-  );
-
   const direction = isMobile ? 'column-reverse' : 'row';
   const spacing = isMobile ? 3 : 0;
 
+  const apiError = hasNotificationSentApiError && (
```

Comment:
I don't think that it is needed. The complexity should be the same. Try to restore the previous version

Reply:
restore in previous version